packages/app-frontend/src/features/timeline/TimelineView.vue

```diff
@@ -125,11 +127,13 @@ export default {
       updateEventPosition(event, g)
       g.y = 16
       event.g = g
-      if (selectedEvent.value === event) {
-        drawSelectedEvent(event)
-      } else {
-        drawUnselectedEvent(event)
-      }
+      setTimeout(() => {
```

Comment:
Why the timeout?

Reply:
ah yeah, I forgot about this one and didn't put a comment! I realized that stackedEvents do not appear immediately, so some stacked events would render the first event meaning that one normal and an error would render as normal. I imagine the problem is somewhere else, where the call to `addEvent` is done but I didn't dig more 😅